packages/cyberstorm/src/components/StaleCover/StaleCover.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./StaleCover.module.css";
+import { ReactNode } from "react";
+import { classnames } from "../../utils/utils";
+
+interface Props {
+  children: ReactNode;
+  className?: string;
+  isStale: boolean;
+}
+
+/**
+ * Component for covering children while their content is stale
+ */
+export function StaleCover(props: Props) {
```

Comment:
Nitpick: technically this doesn't cover the component (like e.g. a modal background layer would) since it's rendered below the component. `StaleIndicator` or `StalenessIndicator` oslt could be more descriptive.